src/components/tokens/components/TransferResult.js

```diff
@@ -2,19 +2,35 @@ import React from 'react';
 import { Avatar,Icon,Button,Card } from 'antd';
 
 let Preview = ({
-  modals,
+  modal,
   }) => {
+  const {result} = modal
+  console.log(result)
   return (
       <Card title="Result">
         <div className="p25 text-center">
-          <Icon className="fs60" type="check-circle"></Icon>
-          <div className="fs20 color-grey-900">
-            Send Completed
-          </div>
-          <div className="fs14 color-grey-900">
-            You have successfully sent 1000 LRC (USD 5880)
-          </div>
-        </div> 
+          {result.error &&
+            <div>
+              <div className="fs14 color-grey-900">
+                {"Your hava failed sent "+result.extraData.amount+" "+result.extraData.tokenSymbol+" cause: "+result.error}
+              </div>
+            </div>
+          }
+          {!result.error &&
+            <div>
+              <Icon className="fs60" type="check-circle"></Icon>
+              <div className="fs20 color-grey-900">
+                Send Completed
+              </div>
+              <div className="fs14 color-grey-900">
+                {"You have successfully sent "+result.extraData.amount+" "+result.extraData.tokenSymbol+" (≈USD "+result.extraData.worth+")"}
```

Comment:
{`You have successfully sent ${result.extraData.amount} ${result.extraData.tokenSymbol} " (≈USD ${result.extraData.worth})`}   use `` syntax to concat string